config.yml

```diff
@@ -3,8 +3,8 @@ plugins:
 
 settings:
   log-level: debug
-  intercept-log: /var/log/postgres-proxy/intercept.log
-  general-log: /var/log/postgres-proxy/general.log
+  intercept-log: logs/intercept.log
```

Comment:
made the default logs a local directory (within the project) so the user doesn't have to go searching on their computer but can see the logs immediately in their IDE